pkg/reconciler/trigger/trigger.go

```diff
@@ -207,6 +240,37 @@ func (r *Reconciler) reconcile(ctx context.Context, t *v1alpha1.Trigger) error {
 	return nil
 }
 
+func (r *Reconciler) triggerMarkedReadyByDependency(dependencyObjRef corev1.ObjectReference, ctx context.Context, t *v1alpha1.Trigger) error {
```

Comment:
```suggestion
func (r *Reconciler) propagateDependencyReadiness(ctx context.Context, t *v1alpha1.Trigger, dependencyObjRef corev1.ObjectReference) error {
```

The convention I normally see is that context is the first argument.